README.md

```diff
@@ -62,17 +72,48 @@ Username: postgres
 Password: (not necessary)
 ```
 
-6) Ensure the database `dbautoscheduler` has been created:
+7) Ensure the database `dbautoscheduler` has been created:
 Using psql: Type `\l` in the psql prompt and see if `dbautoscheduler` is in the list.
 Using pgAdmin: Click the name of the server you created, and see if `dbautoscheduler` is in the tree menu.
-7) If you'd like to get the current project running, you can clone the repository with `git clone https://github.com/gannonprudhomme/AutoScheduler/` and run the steps below.
+
+8) Navigate to the frontend directory: `./autoscheduler/frontend/src/` and run `npm install`
+    - This is for TypeScript compilation and testing.
+    
+9) Before making code changes, please go to the github project [wiki](https://github.com/aggie-coding-club/Rev-Registration/wiki) and follow the Project Setup and Install section.
 
 ## Running
 
 Before running any commands, if you're not running in the virtual environment(you should see `(env)` somewhere in your current terminal line), run `source env/bin/activate` or `./env/Scripts/activate`.
 
-You will have to make and apply migrations before running the server (and whenever our models are changed). To generate the files that django uses to apply migrations, run `./manage.py makemigrations`, and to then apply these to the database, run `./manage.py migrate`.
+**Server:**
+- You will have to make and apply migrations before running the server (and whenever our models are changed). To generate the files that django uses to apply migrations, run `./manage.py makemigrations`, and to then apply these to the database, run `./manage.py migrate`.
+    - `manage.py` can be found in the `./autoscheduler/` directory.
+
+- After migrating, you can run the server with `./manage.py runserver`. When running for the first time, you will have to configure `autoscheduler/autoscheduler/config/postgres-info.json` with your postgres username and password. This file will be automatically created if you run `./manage.py runserver`.
```

Comment:
```suggestion
- After migrating, you can run the server with `./manage.py runserver`. When running for the first time, you will have to configure `autoscheduler/autoscheduler/config/postgres-info.json` with your postgres username and password. This file will be automatically created if you run `./manage.py runserver`.
     - This starts a webserver that's listening on port 8000, so you can navigate to `localhost:8000/` and you should see the landing page
          - Note: you must have run `npm run dev` from below
```

Adds two lines about navigating to localhost:8000 (since this is really what lets you do it)